ember-model.js

```diff
@@ -340,29 +340,29 @@ Ember.ManyArray = Ember.RecordArray.extend({
   },
 
   replaceContent: function(index, removed, added) {
-    added = Ember.EnumerableUtils.map(added, function(record) {
+    added = added.map(function(record) {
       return record._reference;
     }, this);
 
     this._super(index, removed, added);
   },
 
-  _contentWillChange: function() {
+  _contentDidChange: function() {
```

Comment:
They've been merged. We did branch off master.

They were merged without running `grunt release` so the changes are in the `packages` directory but not in the concated release file that we actually use through bower.